doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -335,7 +338,16 @@ The optional configuration for monitoring parameters is installed by package xr
 
 The central monitoring client may be configured as specified in the [UG-CS](#Ref_UG-CS).
 
-### 2.10 Post-Installation Checks
+### 2.10 Pre-configuration for Registration Web Service
+
+The optional configuration for registration web service is installed by package xroad-centralserver-registration-service. The package is included in the central server installation by default.
```

Comment:
This sentence is a little bit hard to understand. Why it's said that the configuration for registration web service is optional? The registration web service can be installed 1) on the same host with the Central Server or 2) on a separate host. In case a separate host is used, the configuration isn't required on the Central Server. If that's the meaning of this part, then I'd rephrase it:

> The registration wed service is installed by package `xroad-centralserver-registration-service`. The registration web service can be installed on the same host with the central server or on a separate host. The package is included in the central server installation by default.